src/components/atoms/Button/Button.tsx

```diff
@@ -29,6 +30,7 @@ export const AppButton: React.FC<ButtonProps> = ({
   isLink = false,
   linkTo,
   disabled = false,
+  newTab = false,
```

Comment:
```suggestion
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/f038fa7f07dd402677ba46b787847df727270978